mat64/format.go

```diff
@@ -11,13 +11,14 @@ import (
 
 // Format prints a pretty representation of m to the fs io.Writer. The format character c
 // specifies the numerical representation of of elements; valid values are those for float64
-// specified in the fmt package, with their associated flags. In addition to this, a '#' for
-// all valid verbs except 'v' indicates that zero values be represented by the dot character.
-// The '#' associated with the 'v' verb formats the matrix with Go syntax representation.
+// specified in the fmt package, with their associated flags. In addition to this, a space
+// preceding a verb indicates that zero values should be represented by the dot character.
 // The printed range of the matrix can be limited by specifying a positive value for margin;
 // If margin is greater than zero, only the first and last margin rows/columns of the matrix
 // are output.
-func Format(m Matrix, margin int, dot byte, fs fmt.State, c rune) {
+//
+// Format will not provide Go syntax output.
```

Comment:
After this comment, I still don't know what prefix is for, and I don't know how I get a fmt.State. I think the resolution to prefix has  something to do with my "preceeding a verb" question. I don't expect an explanation of fmt.State, but a pointer would be nice.